apps/web/server/lib/[user]/getServerSideProps.ts

```diff
@@ -132,6 +133,7 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
   const [user] = usersInOrgContext; //to be used when dealing with single user, not dynamic group
 
   const profile = {
+    id: user.id,
```

Comment:
passing id here to get user's avatar url (needed for OG image generation in generateMetadata function)